src/test/groovy/uk/gov/hmrc/jenkinsjobs/JobsSpec.groovy

```diff
@@ -12,7 +12,7 @@ class JobsSpec extends Specification {
     @Unroll
     void 'test script #file.name'(File file) {
         when:
-        DslScriptLoader.runDslEngine file.text, new MemoryJobManagement()
+        new DslScriptLoader(new MemoryJobManagement()).runScript file.text
 
```

Comment:
Can't make sense of this test change. @arturopala Can you please me a brief on this?

Reply:
it's consequence of jenkins-job-builders version update

Reply:
👍